src/components/TradingViewChart/datafeed.tsx

```diff
@@ -124,8 +124,16 @@ export const getCandlesApi = (
     }`,
   )
 }
-const checkIsUSDToken = (chainId: ChainId | undefined, currency: Currency | undefined) => {
-  if (currency?.isNative || !chainId) {
+
+const isNativeToken = (chainId: ChainId | undefined, currency: Currency | undefined) => {
+  if (!currency || !chainId) {
+    return false
+  }
+  return currency.isNative || WETH[chainId].address.toLowerCase() === currency.address.toLowerCase()
+}
+
+const isUSDToken = (chainId: ChainId | undefined, currency: Currency | undefined) => {
+  if (isNativeToken(chainId, currency) || !chainId || currency?.isNative) {
```

Comment:
`isNativeToken` func has already check `currency.isNative` => dont need check it again

Reply:
my bad. ignore

Reply:
no, I need it also check for wrapped token, to enable chart for wrapped native